src/app/wallet/proposals/proposal-details/proposal-details.component.ts

```diff
@@ -43,25 +43,43 @@ export class ProposalDetailsComponent implements OnInit, OnDestroy {
       showLabels: false,
       donut: true,
       legend: {
-        maxKeyLength: 35,
+        maxKeyLength: 100,
         margin: {
           left: 0,
           right: 0,
           top: 5,
           bottom: 5
         },
-        // padding: {
-        //   top: 0,
-        //   bottom: 0,
-        //   right: 5,
-        //   left: 5
-        // }
+        padding: {
```

Comment:
Please do not change this, ie: leave it commented out or better yet, remove the padding object here altogether. Using an object for padding results in errors in the console. The correct way to set padding appears to be via a number value (rather than an object value).

Reply:
Hye @zaSmilingIdiot I am not which error you are getting tho the padding helps in letting the text to start after the circle point. If remove that then the  initial text will be hide and looks messy .

Thanks

Reply:
You can generate the error by ensuring that the JS console is open, and load the Proposals page. An error is generated for each graph. Additional errors can be generated by resizing the window.

Have a look over here: http://krispo.github.io/angular-nvd3/#/donutChart
Change the radio button from 'Basic' to 'Extended' and then expand: cahrt -> legend. The 'padding' value accepts a numerical value not an object. If the padding is required, then try to replace it with a number rather than an object. I suspect that if yo uneed further fine grain control of the padding values that you may need to use the chart -> styles -> object, or an alternative solution (css styles for the svg elements? ).

Reply:
yeah correct !! I missed the console :P :D